src/ipfs.io/docs/getting-started/index.html

```diff
@@ -42,6 +42,11 @@
 
 <div class="alert alert-info">
 Note the hash there may differ. If it does, use the one you got.
+
+Your `peer identity` is your unique identity on the `ipfs` network,
+so you may not want to publish it on the net together with your `real`
+identity, since then all files you publish can potentially be tracked
+back to you.
```

Comment:
I disagree with this wording, and it also doesn't capture the fact that peers learn each others' ip addresses, which is the same security concern. Perhaps better:

> Your `peer identity` is your unique identity on the `ipfs` network. It uniquely identifies your node.